bin/growpart

```diff
@@ -744,6 +744,14 @@ has_cmd() {
 	command -v "${1}" >/dev/null 2>&1
 }
 
+cmd_is_busybox() {
```

Comment:
Did you test this? I don’t think it will work. 

Command -v will probably output ‘/bin/blkid’. 

You might be able to then test:


    out=$(command -v “$1”) && [ “$out” -ef /bin/busybox ]
    

I’m not sure if -ef works on symlink or not. Ideally you would not depend on busybox being /bin/busybox but rather anywhere in PATH.

Reply:
I did so some tested a few days and thought it was working, but on a second look you're right. I don't know how my previous testing didn't spot any issues.

I have reworked it not and have tested it on Alpine against both Busybox's and util-linux's versions of blkid and also with Busybox's and coreutils' versions of basename and readlink and have verified (after adding "-x" to the shebang) that the code is definately following any softlink.

I also added the use of "basename" to address your point about busybox being located not only in /bin.